migrations/1604516429383-CreatePersistentSignedOrder.ts

```diff
@@ -0,0 +1,42 @@
+import { OrderEventEndState } from '@0x/mesh-rpc-client';
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const OrderEventEndStateStrings = Object.keys(OrderEventEndState).filter(x => isNaN(parseInt(x, 10)));
+
+export class CreatePersistentSignedOrder1604516429383 implements MigrationInterface {
+    // tslint:disable-next-line
+    public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
do we also create the indexes here?
Everytime we ran with `synchronize` true our indexes would be deleted >< I guess since they weren't explicitly defined.

But we'd want indexes on:
```
maker_address
maker_asset_data
taker_asset_data
fee_recipient_address
```